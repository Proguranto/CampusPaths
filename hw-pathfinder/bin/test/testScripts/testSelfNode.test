# Find a path with Dijkstra's algorithm when
# the start node = dest node.

CreateGraph graph9

AddNode graph9 S

FindPath graph9 S S
