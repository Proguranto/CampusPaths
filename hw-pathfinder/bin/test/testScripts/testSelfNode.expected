# Find a path with Dijkstra's algorithm when
# the start node = dest node.

created graph graph9

added node S to graph9

path from S to S:
S to S with weight 0.000
total cost: 0.000
